### ISSUE TITLE
FIX 4.2 error in message name AllocationInstructionAck and MsgType(35) values J and P

### ISSUE THREAD

**Issue by kleihan** [FIXTradingCommunity/orchestrations#28]
From [FIXimate](https://fiximate.fixtrading.org/en/FIX.Latest/msg19.html):
> The Allocation Instruction message provides the ability to specify how an order or set of orders should be subdivided amongst one or more accounts. In versions of FIX prior to version 4.4, this same message was known as the Allocation message. Note in versions of FIX prior to version 4.4, the allocation message was also used to communicate fee and expense details from the Sellside to the Buyside. This role has now been removed from the Allocation Instruction and is now performed by the new (to version 4.4) Allocation Report and Confirmation messages.

The FIX 4.2 Orchestra file has three errors related to this change:

- Message name is from FIX 4.4: `<fixr:message name="AllocationInstructionAck" id="24" msgType="P" ...`
==> change to `<fixr:message name="AllocationAck" id="24" msgType="P" ...`
- Code name for MsgType(35)=J is from FIX 4.4: `<fixr:code name="AllocationInstruction" id="35019" value="J" ...`
==> change to `<fixr:code name="Allocation" id="35019" value="J" ...`
- Code name for MsgType(35)=P is from FIX 4.4: `<fixr:code name="AllocationInstructionAck" id="35024" value="P" ...`
==> change to `<fixr:code name="Allocation ACK" id="35024" value="P" ...` (note the space before "ACK")

**Comment by kleihan** [comment]
See also SPEC-2665

**Comment by kleihan** [comment]
Correction to description above for code name for MsgType(35)=P. This is the symbolic name and must *not* include spaces. The synopsis (documentation) can have spaces but is already correct: 
```xml
<fixr:code name="AllocationInstructionAck" id="35024" value="P" sort="23" added="FIX.2.7">
	<fixr:annotation>
		<fixr:documentation purpose="SYNOPSIS">
                  Allocation ACK
               </fixr:documentation>
	</fixr:annotation>
</fixr:code>
```

**Comment by kleihan** [comment]
Resolved with a70274f392cbb28fd776c91a5d07bc2c20b4646a
